frontend/packages/erd-core/package.json

```diff
@@ -6,7 +6,10 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "src/index.ts",
+  "exports": {
+    ".": "./src/index.ts",
+    "./nextjs": "./src/nextjs/index.ts"
+  },
```

Comment:
@liam-hq/erd-core package will export the Provider for Next.js by `@liam-hq/erd-core/nextjs`